pages/storage_and_backup/object_storage/Versioning_Doc/guide.en-gb.md

```diff
@@ -0,0 +1,129 @@
+---
+title: 'Getting Started with Versioning Using OVHcloud APIs'
+excerpt: 'Learn how to enable and manage versioning for your OVHcloud Object Storage buckets using APIs. Future updates will cover the OVHcloud Control Panel.'
+updated: 2024-06-26
+---
+
+## Objective
+
+**This guide explains how to enable and manage versioning for your OVHcloud Object Storage buckets using APIs.**
+
+## Requirements
+
+- A Public Cloud project in your OVHcloud account
+- Access to the OVHcloud Control Panel
+- An S3 user already created
+- AWS CLI installed and configured
+
+## Instructions
+
+### Concept
+
+Versioning in Object Storage allows you to keep multiple variants of an object in the same bucket. This feature helps preserve, retrieve, and restore every version of every object stored in your buckets, making it easier to recover from unintended user actions or application failures. By default, versioning is disabled on buckets, and you must explicitly enable it.
+
+### General Information
+
+An Object Storage bucket can be in one of three states:
+
+1. **Unversioned** (default state): No versions are kept for the objects.
+2. **Versioning-enabled**: Multiple versions of each object are kept.
+3. **Versioning-suspended**: Versions are not created for new objects, but existing versions are retained.
+
+> [!warning]
+> Once versioning is enabled, it cannot be disabled; it can only be suspended.
+
+
+![Archive](images/versionning.png)
```

Comment:
All images links must end with `{.thumbnail}`
The alt text must also be coherent and describing the image. Here "Archive" doesn't seem adequate.

This comment is also valid for all other images in this guide.